datahub-web-react/src/app/shared/RoutedTabs.tsx

```diff
@@ -19,6 +20,13 @@ interface Props extends TabsProps {
     onTabChange?: (selectedTab: string) => void;
 }
 
+const RoutedTabsStyle = styled.div`
+    max-height: 100%;
```

Comment:
this has impacts across the application. it is widely used. i would recomend we pls try to avoid changing this if possible. why is this needd?

Reply:
we can remove max-height it will work same but can't remove flex property because it creating container for inside item and when there is more items it help us to add scroll inside this container